### DISCUSSION TITLE
Pros & Cons - HydraAdmin vs OpenApiAdmin

### DISCUSSION THREAD

**Discussion by grichards** [api-platform/core#6129] [Q&A]
API Platform Core provides docs for both formats out of the box, correct?

API Platform Admin provides both HydraAdmin and OpenApiAdmin components, correct?

How might one go about choosing which to use with an API Platform backend?

Is anyone willing to share their thoughts about the differences, considerations, pros, cons, etc?

Thank you!
Gabe

**Comment by soyuka** [comment]
I never tried the OpenApi admin module, but both work on top of [React Admin](https://marmelab.com/react-admin/) and they are parsed by the [api-doc-parser](https://github.com/api-platform/api-doc-parser/). I'd say that the Hydra one is there for a longer period, and is probably more mature. I also tend to prefer RDF formats like [Hydra](https://www.hydra-cg.com/) for these things but in the end it's a matter of personal preferences.
